job-dsls/jobs/prod/prod_rhba_properties_generator.groovy

```diff
@@ -73,7 +73,7 @@ pipelineJob("${folderPath}/rhba-properties-generator") {
     parameters {
         booleanParam("IS_PROD", true, "it defines if the properties file is for prod or not")
         stringParam("BRANCH_NAME", "main", "the branch the nightly was triggered for")
-        stringParam("REPO_URL", "http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/rhba-main-nightly", "Prod possibility is http://rcm-guest.app.eng.bos.redhat.com/rcm-guest/staging")
+        stringParam("REPO_URL", "http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/rhba-main-nightly", "Prod possibility is http://download.devel.redhat.com/staging")
```

Comment:
Isn't it `http://download.devel.redhat.com/rcm-guest/staging`? Just asking, since I saw it somewhere..

Reply:
http://download.devel.redhat.com/rcm-guest/staging/ is correct

Reply:
```suggestion
        stringParam("REPO_URL", "http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/rhba-main-nightly", "Prod possibility is http://download.devel.redhat.com/rcm-guest/staging/")
```